packages/core/src/services/commits/create.ts

```diff
@@ -0,0 +1,11 @@
+import database from '$core/client'
+import { Commit, commits } from '$core/schema'
+
+export default async function createCommit(tx = database) {
+  return (
```

Comment:
Can we always return Results in services? This way is easier to concatenate

Reply:
yup